betterconf/caster.py

```diff
@@ -56,6 +57,43 @@ def cast(self, val: str) -> typing.Union[int, typing.NoReturn]:
             raise ImpossibleToCastError(val, self)
 
 
+class FloatCaster(AbstractCaster):
+    def cast(self, val: str) -> typing.Union[float, typing.NoReturn]:
+        val = val.replace(",", ".")
+        try:
+            as_float = float(val)
```

Comment:
I've looked at `locale` module and it has `atof` function that converts strings to float, and **is** aware of locale-specific decimal point characters. You can view docs for the module [here](https://docs.python.org/3.3/library/locale.html). Also I guess some languages have more weird point characters, so this'd be a good idea.
```suggestion
            as_float = locale.atof(val)
```

Reply:
Looking at your last comment (about tests), I guess just `float` is more acceptable here because it doesn't care about current local (so, we are able to convert both _point_ (3.14) and _comma_ (3,14) numbers).